ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host1/l3_edge_multicast.yml

```
@@ -0,0 +1,30 @@
+underlay_routing_protocol: ospf
+underlay_multicast: true
+
+type: spine
+spine:
+  nodes:
+    - name: host1
+      id: 1
+      evpn_role: "none"
+      loopback_ipv4_pool: 1.2.3.4/24
+      bgp_as: 65000
+
+l3_edge:
+  p2p_links_ip_pools:
+    - name: pool1
+      ipv4_pool: 192.168.0.0/24
+
+  p2p_links:
+      # P2P link with underlay multicast(pim-sparse) disabled.
+    - nodes: [host1, peer1]
+      interfaces: [ethernet1, ethernet1]
+      ip: [192.168.0.4/31, 192.168.0.5/31]
+      include_in_underlay_protocol: true
+      underlay_multicast: true
```

```suggestion
      underlay_multicast: false
```
This should be false to `disable` underlay multicast pim-sparse (as mentioned in the comment)

---

added